tools/ci-ts/tests/flux-monitor.test.ts

```diff
@@ -1,338 +0,0 @@
-import { assert } from 'chai'
```

Comment:
Where did this file move to?

Reply:
Good spot. Look like this slipped through the cracks being in the `tools/ci-ts` directory.

I'll readd this and convert the tools/ci-ts directory to using ethers5.